python/felis/datamodel.py

```diff
@@ -49,7 +58,7 @@
 CONFIG = ConfigDict(
     populate_by_name=True,  # Populate attributes by name.
     extra="forbid",  # Do not allow extra fields.
-    validate_assignment=True,  # Validate assignments after model is created.
+    validate_assignment=False,  # Validate assignments after model is created.
```

Comment:
This is changed because validating after assignment re-executes validation functions, which is undesirable behavior in some cases. So I'm turning this off for now. It isn't clear that we would ever want a loaded schema to be modified by the user anyways.